src/api/api_epg.c

```diff
@@ -475,6 +475,10 @@ api_epg_grid
 
   epg_query_free(&eq);
   free(lang);
+  if (eq.lang)        free(eq.lang);
```

Comment:
This probably belogs to epg_query_free(), doesn't ?

Also, it seems that epg_query() does such free calls.

Reply:
Yes, you're completely right.
I think it's because I saw the alloc and no free when I was adding my filter on the other PR so added them in.

Let's scrap this PR and I'll fix up my #1050 to move my free in to epg_query.